instance/tests/models/test_instance.py

```diff
@@ -407,6 +408,25 @@ def test_provision(self, os_server_manager, mock_run_playbook, mock_set_dns_reco
         self.assertEqual(mock_server_reboot.call_count, 1)
 
     @patch_os_server
+    @patch('instance.models.server.openstack.create_server')
+    @patch('instance.models.server.OpenStackServer.sleep_until_status')
+    @patch('instance.models.server.OpenStackServer.reboot')
+    @patch('instance.models.instance.gandi.set_dns_record')
+    @patch('instance.models.instance.OpenEdXInstance.run_playbook')
+    def test_provision_failed(self, os_server_manager, mock_run_playbook, mock_set_dns_record, mock_server_reboot,
+                              mock_sleep_until_status, mock_openstack_create_server):
+        """
+        Run provisioning sequence
+        """
+        mock_run_playbook.return_value = (['log'], 1)
+        mock_openstack_create_server.return_value.id = 'test-run-provisioning-server'
+        os_server_manager.add_fixture('test-run-provisioning-server', 'openstack/api_server_2_active.json')
+
+        instance = OpenEdXInstanceFactory(sub_domain='run.provisioning', status=Server.BOOTED)
+        result = instance.provision()
+        self.assertEqual(result[1], ['log'])
```

Comment:
True, I think that was not supposed to be committed. I went ahead and finished implementing this test anyways. Thanks for reporting.